[Webview UI toolkit deprecation] AttachAcrToCluster, DraftDeployment/Dockerfile/Workflow Panel component removal

tejhan opened the pull request:
This PR specifically removes elements from the AttachAcrToCluster & DraftDeployment/Dockerfile/Workflow commands.

**.vsix for testing:** [vscode-aks-tools-1.6.0-dep3-2.vsix.zip](https://github.com/user-attachments/files/19069996/vscode-aks-tools-1.6.0-dep3-2.vsix.zip)


This is broken up as part of incremental PR's to make review & testing smoother.

For future reference:
- .secondary-button replaces VSCodeButton's appearance="secondary"
- .icon-button replaces VSCodeButton's appearance="icon"
- input[type="text"] replaces VSCodeTextInput
- input[type="checkbox"] replaces VSCodeCheckbox
- input[type="radio"] replaces VSCodeRadio
- Default "< hr >" element now has identical styling to VSCodeDivider
-  Default "< a >" element also has identical styling to VSCodeLink

ReinierCC commented:
AttachAcrToCluster worked well. ✅ 

For DraftDeployment after clicking create there's some text fields that don't get greyed out. As well the selected deployment option no longer appears.

Before:
<img width="652" alt="Screenshot 2025-02-21 at 11 03 43 AM" src="https://github.com/user-attachments/assets/fd442544-6821-4791-9605-2fa383733286" />

After:
<img width="594" alt="Screenshot 2025-02-21 at 11 02 34 AM" src="https://github.com/user-attachments/assets/2c062d2a-2014-4275-8b7c-a918d2654d81" />

Similar Issue with Draft Dockerfile & Workflow
<img width="589" alt="Screenshot 2025-02-21 at 11 09 04 AM" src="https://github.com/user-attachments/assets/f866e338-b0ec-4255-98dd-bdacb567d440" />

<img width="613" alt="Screenshot 2025-02-21 at 11 11 33 AM" src="https://github.com/user-attachments/assets/e3d3c213-e9fe-4e38-9183-aa51d2ec1ba1" />

tejhan commented:
> AttachAcrToCluster worked well. ✅
> 
> For DraftDeployment after clicking create there's some text fields that don't get greyed out. As well the selected deployment option no longer appears.
> 
> Before: <img alt="Screenshot 2025-02-21 at 11 03 43 AM" width="652" src="https://private-user-images.githubusercontent.com/30391816/415732431-fd442544-6821-4791-9605-2fa383733286.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwODc0MDYsIm5iZiI6MTc0MTA4NzEwNiwicGF0aCI6Ii8zMDM5MTgxNi80MTU3MzI0MzEtZmQ0NDI1NDQtNjgyMS00NzkxLTk2MDUtMmZhMzgzNzMzMjg2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDExMTgyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQwOWI5YjE1MzQ4YzYwNTU0NDg1OWM4ZWE2YjZmZjJiN2FmNzIzMjg4MjlmMWMyOGM3ODFkOWE4YjVmODJlZTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.UFNZX-svdKdCWcAedWipbeqg1MhcCMStNWy3x8xbELk">
> 
> After: <img alt="Screenshot 2025-02-21 at 11 02 34 AM" width="594" src="https://private-user-images.githubusercontent.com/30391816/415731931-2c062d2a-2014-4275-8b7c-a918d2654d81.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwODc0MDYsIm5iZiI6MTc0MTA4NzEwNiwicGF0aCI6Ii8zMDM5MTgxNi80MTU3MzE5MzEtMmMwNjJkMmEtMjAxNC00Mjc1LThiN2MtYTkxOGQyNjU0ZDgxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDExMTgyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIyYWNkZGQ3NzhiYzM4ZGE2MmRkY2ZjYjY5ZjAxMmNhYTE5ZjRlN2YwY2I5ZGUyNTJkZGQ3NTBjZTQ3MmYwYWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.K95ZrsJrbaFeaWI6xiE9BHUdbVV-1QHPaROTD9jUw3I">
> 
> Similar Issue with Draft Dockerfile & Workflow <img alt="Screenshot 2025-02-21 at 11 09 04 AM" width="589" src="https://private-user-images.githubusercontent.com/30391816/415735028-f866e338-b0ec-4255-98dd-bdacb567d440.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwODc0MDYsIm5iZiI6MTc0MTA4NzEwNiwicGF0aCI6Ii8zMDM5MTgxNi80MTU3MzUwMjgtZjg2NmUzMzgtYjBlYy00MjU1LTk4ZGQtYmRhY2I1NjdkNDQwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDExMTgyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEyZmVlYmYzOGQ0YzhiMmVhOTVlNjNmMzRmNDM3NTIwMDY2ZDA4ZWVjMTg4ZGIwM2I5OTQ3ODJmNGQyNWZjNTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Eaz_-b0UzXe-q-6vsh0B0sak5F29rpRlSD0B7DdiPek">
> 
> <img alt="Screenshot 2025-02-21 at 11 11 33 AM" width="613" src="https://private-user-images.githubusercontent.com/30391816/415736174-e3d3c213-e9fe-4e38-9183-aa51d2ec1ba1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwODc0MDYsIm5iZiI6MTc0MTA4NzEwNiwicGF0aCI6Ii8zMDM5MTgxNi80MTU3MzYxNzQtZTNkM2MyMTMtZTlmZS00ZTM4LTkxODMtYWE1MWQyZWMxYmExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDExMTgyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwZWVlZjgyMzcxNTMyYjUwYjllMWExM2I2MzM4YjQyNmVlZTQ3M2IwNzNiN2M3MmU4YWI5YjYyZDE2YzU5NTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.M_YYwgyQiGB3h2lqxUDwcPgSInMAT1s-ctbHRJ8P8Y8">

It seems there was no default styling of disabled text inputs, so I've added that, and the checkbox actually did retain the selection but is hard to view with the default color scheme on a black background, so I've changed the checkboxes to be theme based as well, so both checkboxes & text fields should now properly visually indicate their disabled state.

ReinierCC reviewed (approved):
Looks good! Tested on Windows.
